src/app/center-bottom-content/components-bottom/report-builder/workflow/workflow.component.html

```diff
@@ -1,31 +1,101 @@
-              <!-- Text -->
-              <div style="margin-left: 40px" *ngIf="step.value.type === 'text'">
-                <input type="text" formControlName="text"> 
-              </div>
-            </div>
-            <button (click)="nextStep(i); onContinue(workflowForm.value)">Next</button>
-          </div>
-        </div>  
-      </div>
-    </div>
-    <br><button *ngIf="finalStep" type="button" (click)="finishedWorkflow(workflowForm.value)">Finish</button>
-  </form>
-</div>
+
+
+<div class="workflow-wrapper">
+	<h3>{{workflow.title}}</h3>
+	
+	<form [formGroup]="workflowForm" (ngSubmit)="onContinue(workflowForm.value)" #data>
+	
+		<!-- USWDS Step Indicator -->
+		<div class="usa-step-indicator usa-step-indicator--counters-sm" aria-label="progress" formArrayName="steps">
+			<ol class="usa-step-indicator__segments">
+	
+				<li class="usa-step-indicator__segment usa-step-indicator__segment--complete" 
+					*ngFor='let step of getSteps(workflowForm)| slice:0:stepsCompleted+1; let i = index'>
+	
+					<span class="usa-step-indicator__segment-label">
+						Step {{i+1}}:
+						<span class="usa-sr-only">completed</span>
+					</span>
+				</li>
+	
+			</ol>
+		</div>
+	
+	
+	
+		<!-- Form -->
+		<div formArrayName="steps" class="horizontal-workflow-steps">
+	
+			<div *ngFor='let step of getSteps(workflowForm)| slice:0:stepsCompleted+1; let i = index' class="workflow-step" [ngClass]="{'completed': stepsCompleted > i}">
+	
+				<!-- Step title -->
+				<h6>{{ step.value.label}}</h6>
+
+
+				<div formGroupName="{{i}}">
+	
+					<div formArrayName="options">
+	
+						<!-- Loop for each step -->
+						<div
+							*ngFor="let option of getOptions(step); let j = index" formGroupName="{{j}}">
+	
+							<!-- Checkboxes -->
+							<div class="workflow-radio-group" *ngIf="step.value.type === 'checkbox'">
+								<input type="checkbox" id="workflowRadio{{j}}" name="workflowRadioSelection" formControlName="selected" [checked]="option.value.selected" (change)="onCheckboxChange(option.value, step.value)">
+								<label class="" for="workflowRadio{{j}}" >{{option.value.text}}</label>
+							</div>
+	
+							<!-- Subscription -->
+							<div *ngIf="step.value.type === 'subscription'">
+								<input type="hidden" formControlName="selected" />
+								<label>
+									{{option.value.text}}
+								</label>
+							</div>
+	
+							<!-- Text -->
+							<div *ngIf="step.value.type === 'text'">
+								<input type="text" class="usa-input" formControlName="text" />
+							</div>
+						
+						</div>
+	
+					</div>
+	
+				</div>
+	
+	
+	
+				<!-- Next Button -->
+				<div>
+					<button (click)="nextStep(i); onContinue(workflowForm.value)" class="usa-button mtop-xs" [disabled]="stepsCompleted > i">Next</button>
+				</div>
+	
+	
+									
+			</div>
+	
+			<!-- Finish Button -->
+			<div class="workflow-step" *ngIf="finalStep">
+				<div>
+					<label>
+						You've completed this 
+						<br/>
+						{{workflow.title}}. 
```

Comment:
Recommend changing this to "You've completed this {{workflow.title}} workflow." Sometimes it will say things like "You've completed this Fire Hydrology." which doesn't make as much sense.